app/src/main/res/layout/activity_audio_detail.xml

```diff
@@ -0,0 +1,170 @@
+                        android:contentDescription="@string/description_back_to_timeline"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent"
+                        app:srcCompat="@drawable/ic_backspace" />
+
+                    <ImageView
+                        android:layout_width="@dimen/dp_24"
+                        android:layout_height="@dimen/dp_24"
+                        android:contentDescription="@string/description_vote_for_post"
+                        android:padding="@dimen/dp_2"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintTop_toTopOf="parent"
+                        app:srcCompat="@drawable/ic_vote" />
+
+                </android.support.constraint.ConstraintLayout>
+
+                <View
+                    android:id="@+id/view_header"
+                    android:layout_width="0dp"
+                    android:layout_height="@dimen/dp_2"
+                    android:background="@color/color_grey_200"
+                    app:layout_constraintTop_toBottomOf="@id/constraint_header" />
+
+                <TextView
+                    android:id="@+id/text_content"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="@dimen/dp_8"
+                    android:layout_marginStart="@dimen/dp_8"
+                    android:layout_marginTop="@dimen/dp_8"
+                    android:background="@drawable/bg_timeline_conversation"
+                    android:gravity="center"
+                    android:maxLines="3"
+                    android:padding="@dimen/dp_4"
+                    android:textColor="@android:color/black"
+                    android:textStyle="bold"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/view_header" />
+
+                <View
+                    android:id="@+id/view_audio"
+                    android:layout_width="0dp"
+                    android:layout_height="@dimen/dp_2"
+                    android:layout_marginTop="@dimen/dp_8"
+                    android:background="@color/color_grey_200"
+                    app:layout_constraintTop_toBottomOf="@id/text_content" />
+
+                <com.google.android.exoplayer2.ui.SimpleExoPlayerView
+                    android:id="@+id/exo_audio"
+                    android:layout_width="0dp"
+                    android:layout_height="240dp"
+                    android:layout_marginTop="@dimen/dp_8"
+                    android:transitionName="@string/transition_audio"
```

Comment:
WARNING: [UnusedAttribute] Attribute `transitionName` is only used in API level 21 and higher (current min is 15)
 This check finds attributes set in XML files that were introduced in a version newer than the oldest version targeted by your application (with the `minSdkVersion` attribute).

This is not an error; the application will simply ignore the attribute. However, if the attribute is important to the appearance of functionality of your application, you should consider finding an alternative way to achieve the same result with only available attributes, and then you can optionally create a copy of the layout in a layout-vNN folder which will be used on API NN or higher where you can take advantage of the newer attribute.

Note: This check does not only apply to attributes. For example, some tags can be unused too, such as the new `<tag>` element in layouts introduced in API 21.